torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16744,9 +16793,20 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
     ElementwiseUnaryPythonRefInfo(
         "_refs.round",
         torch_opinfo_name="round",
+        aliases=('_refs.special.round',),
         # Fails on int32
         # https://github.com/pytorch/pytorch/issues/85258
         supports_nvfuser=False,
+        skips=(
+            # RuntimeError: "round_vml_cpu" not implemented for 'Short', 'Int', 'Long', 'Char', 'Byte'
```

Comment:
Why did these failures start when the alias was added? 

According to the documentation integer inputs should just return a copy of themselves:

https://pytorch.org/docs/master/generated/torch.round.html

Reply:
Filed an issue: https://github.com/pytorch/pytorch/issues/86326

I'm assuming we still want to rely on `test_variant_consistency_eager` to check the aliases and not check them extensively here even though it means having poor coverage for now (due to the above issue), correct?